trendico/models.py

```diff
@@ -1,3 +1,48 @@
 from django.db import models
-
+from datetime import datetime, timedelta
 # Create your models here.
+CATEGORY_CHOICES = (
+    ('L', 'Laptop'),
+    ('T', 'Tab'),
+    ('C', 'Camera'),
+    ('H', 'Headphone')
+)
+class Product(models.Model):
+    product_name=models.CharField(max_length=50)
```

Comment:
this field should be only `name` rather than product_name to avoid repetation of product word